site/client/global-entity/GlobalEntityControl.tsx

```diff
@@ -236,13 +236,9 @@ export class GlobalEntityControl extends React.Component<
     @action.bound private onChange(
         newEntities: ValueType<GlobalEntitySelectionEntity>
     ) {
-        if (newEntities == null) return
+        const entities: GlobalEntitySelectionEntity[] = asArray(newEntities)
 
-        const entities: GlobalEntitySelectionEntity[] = isMultiValue(
-            newEntities
-        )
-            ? Array.from(newEntities)
-            : [newEntities]
+        if (entities.length === 0) return
```

Comment:
I don't know if calling `this.setSelectedEntities` here would be the right thing to do here - I'm just mimicking the old behavior with this early return here.

Reply:
Yeah, should remove this early return and pass the empty array directly onto `setSelectedEntities`